docs/installation/config/openzaak_config.rst

```diff
@@ -253,10 +213,93 @@ Register the required channels:
 
     python src/manage.py register_kanalen
 
+Registering webhooks
+====================
+
+Open Zaak
+---------
+
+As discussed earlier Open Zaak does not require any webhook subscriptions, for now it publishes
+notifications, but doesn't consume them.
+
+Open Notificaties
+-----------------
+
+Open Notifications uses Open Zaak Authorization API, therefore it should subscribe to changes in the
+``autorisaties`` channel. Hence Open Notificaties consumes itself and treats itself as an external service,
+which required configuring related credentials.
+
+8. Configure the credentials for the Open Notificaties API (so Open
+   Notificaties can access itself):
+
+   a. Navigate to **Configuratie > Notificatiescomponentconfiguratie**
+   b. Click on the plus icon besides **Service voor notificaties-api** dropdown.
+   c. Fill out the form **Service toevoegen**:
+
+      - **Label**: *For example:* ``Open Notificaties``
+      - **Type**: Select the option: ``NRC (Notifications)``
+      - **API root url**: the full URL to the Notificaties API root, e.g.
+        ``https://notificaties.gemeente.local/api/v1/``
+      - **Client ID**: We can reuse the existing Client Id from step 3c, since it already has
+        all required permissions to consume notifications. *For example:* ``open-notificaties``
+      - **Secret**: *The same Secret as given in step 3c*
+      - **Authorization type**: Select the option: ``ZGW client_id + secret``
+      - **OAS url**: URL that points to the OpenAPI specification. This is typically
+        ``$api_root`` + ``schema/openapi.yaml``, *for example:*
+        ``https://notificaties.gemeente.local/api/v1/schema/openapi.yaml``
+      - **User ID**: *Same as the Client ID*
+      - **User representation**: *For example:* ``Open Notificaties``
+
+   d. Click **Opslaan**.
+
+9. Configure the Open Notificaties Notificatiescomponent API endpoint (so Open Notificaties
+   receives changes made in the authorisation component of Open Zaak ):
+
+   a. After the previous step 8d you should be on **Configuratie > Notificatiescomponentconfiguratie** page.
```

Comment:
Something I notice is that Github converts the `d` to `iv`.  I wonder if readthedocs does the same

Reply:
As in 8.d is 8.iv

Reply:
I've checked it in RTD and it looks fine:
![image](https://github.com/user-attachments/assets/d37bc0d6-8e67-4e09-a753-bec19006e1df)

You can check the RTD build locally:
```
cd docs
make html
```

And then open `_build/index.html` in the browser